server/tests/helpers.py

```diff
@@ -132,6 +133,7 @@ def create_election(
         {
             "auditName": audit_name or f"Test Audit {datetime.utcnow()}",
             "auditType": audit_type,
+            "ballotPollingType": ballot_polling_type,
```

Comment:
in order to make consistent test data, ballot polling type shouldn't be set if audit type isnt ballot polling